app/views/dashboard/sold.html.erb

```diff
@@ -0,0 +1,79 @@
+<% props = Property.where(user_id: current_user.id) %>
+<% sold_property =  props.where(is_paid: true) %>
+
+<h1 style="text-align:center; margin-top:130px;">Sold Properties</h1>
+
+<section class="property-grid grid">
+  <div class="container">
+    <div class="row">
+      <% sold_property.each do |property| %>
+        <% buyer = User.find(property.buyer_id) %>
+        <div class="box">
+          <div class="row">
+            <div class="col-sm-12 col-md-6" style="float:right;" >
+              <div class="testimonial-img">
+                <%= link_to property, class: "link-a" do  %>
+                  <%= render property %>
+                <% end %>
+              </div>
+            </div>
+            <div class="col-sm-12 col-md-6" style="float:left;">
+              <div class="testimonial-author-box text-center">
+                <h2>Property Details:</h2>
+              </div>
+              <div class="content">
+                <p class="testimonial-text fs-5">
+                  <b>Name: </b><%=  property.name %><br/>
+                  <b>Description: </b><%=  property.description %><br/>
+                  <b>Location: </b><%=  property.address %>,<%=  property.city %><br/>
+                </p>
+              </div>
+              <div class="testimonial-author-box text-center">
+                <h2>Buyer Details:</h2>
+              </div>
+              <div class="content">
+                <p class="testimonial-text fs-5">
+                  <b>Name: </b><%=  buyer.name %><br/>
+                  <b>Email: </b><%=  buyer.email %>
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>            
+      <% end %>
+    </div>
+  </div>
+</section>
+
+
+
+<!-- ======= Footer ======= -->
+<section class="section-footer">
+  <div class="container">
+    <div class="row text-center justify-content-center">
+      <div class="col-sm-12 col-md-4">
+        <div class="widget-a">
+          <div class="w-header-a">
+            <h3 class="w-title-a text-brand">PropEx</h3>
+          </div>
+          <div class="w-body-a">
+            <p class="w-text-a color-text-a">
+              Enim minim veniam quis nostrud exercitation ullamco laboris nisi ut aliquip exea commodo consequat duis
+              sed aute irure.
+            </p>
+          </div>
+          <div class="w-footer-a">
+            <ul class="list-unstyled">
+              <li class="color-a">
+                <span class="color-text-a">Phone .</span> +91 12345 67890
+              </li>
+              <li class="color-a">
+                <span class="color-text-a">Email .</span> <%= mail_to "contact@propex.com", "contact@propex.com", target: '_blank' %>
```

Comment:
Can we make it like...
 
 <span class="color-text-a">Email .</span>
  <%= mail_to "contact@propex.com", "contact@propex.com", target: '_blank' %>

Reply:
- Done